.github/workflows/gradle.yml

```diff
@@ -59,18 +58,14 @@ jobs:
         with:
           # Note, the following glob expression within a filters are ORs.
           filters: |
-            backend:
-              - 'airbyte-!(cdk|integrations|webapp|webapp-e2e-tests)/**'
-              - 'airbyte-integrations/connectors/(destination-jdbc|destination-postgres|source-jdbc|source-postgres)/**'
-              - 'airbyte-config/init/src/main/resources/seed/(source|destination)_definitions.yaml'
-              - 'docker-compose*.yaml'
-              - '(charts|kube)/**'
             build:
               - '.github/**'
               - 'buildSrc/**'
               - 'tools/**'
               - '*.gradle'
               - 'deps.toml'
+              - 'airbyte-config/**'
+              - 'airbyte-featureflag/**'
```

Comment:
I think this one is going away soon

Reply:
I already deleted featureflag yesterday

config, well, I think that story is going to be a longer conversation. It's, um, quite intertwined with both repositories right now